examples/related-visitors/get_related_visitors_200_empty.json

```diff
@@ -0,0 +1,4 @@
+{
+  "relatedVisitors": [
+  ]
+}
```

Comment:
Let's format this 

```suggestion
{
  "relatedVisitors": []
}
```

Reply:
Thank you for catching that. Out of desperation, I thought this was the one causing the lint to fail 😊